tests/integration/targets/aws_s3/tasks/main.yml

```diff
@@ -562,6 +562,60 @@
               - result is not changed
       when: ansible_system == 'Linux' or ansible_distribution == 'MacOSX'
 
+    - name: create an object from static content
```

Comment:
It would be good to test idempotency here:
Run the same 'put' again and see if changed==False

Reply:
Makes sense.